packages/compass-logging/src/provider.ts

```diff
@@ -44,6 +44,8 @@ const LoggerAndTelemetryContext = React.createContext<{
 
 export const LoggerAndTelemetryProvider = LoggerAndTelemetryContext.Provider;
 
+export { createLoggerAndTelemetry } from './ipc-logger';
```

Comment:
You are adding a platform specific dependency export to the provider, this is incorrect, if you are trying to use logger in the render, you should use the hook already exported from there, if you need to inject it to the plugin activate, you should use a locator